script.js

```diff
@@ -0,0 +1,85 @@
+/* 1. Programming Basics
+'Spaghetti', 'Lasagne', 'Beans on toast', 'Porridge', 'Tomato soup', 'Salmon'
+Create an array of the dishes listed above. Loop through the array using a for loop.*/
+
+const dishesArray = [
+  "Spaghetti",
+  "Lasagne",
+  "Beans on toast",
+  "Porridge",
+  "Tomato soup",
+  "Salmon",
+];
+
+// 1.Log out all the dishes.
+
+dishesArray.forEach((dish) => console.log(dish));
+
+//2.Log out a randomly chosen dish (randomly generated  index).
+const randomNumber = Math.floor(Math.random() * dishesArray.length);
+
```

Comment:
Very good, straight to the point! :) 
To take it up a level in a simple and easy way, you could  put the logic that gets a random integer within a certain range into a separate function and use that 💡